projects/uitdatabank/models/event.json

```diff
@@ -115,6 +115,9 @@
         },
         "contributors": {
           "$ref": "./event-contributors.json"
+        },
+        "embedCalendarSummaries": {
```

Comment:
The parameter that you can use on Search API is named `embedCalendarSummaries` (because with the parameter you embed the calendar summaries), but the property that is inside the JSON body of an event or place is called `calendarSummary`

See example below:

*request*
```http
GET https://search.uitdatabank.be/events/?apiKey=0a5a9ce4-7066-11e7-a783-0271c1cbe993&q=id:35f42514-3a33-4709-b266-084aed83f501&embedCalendarSummaries[]=md-text&disableDefaultFilters=true
```

*response*
```http
"member":[
   {
      "@id":"https://io.uitdatabank.be/event/35f42514-3a33-4709-b266-084aed83f501",
      "@type":"Event",
      "calendarSummary":{
         "nl":{
            "text":{
               "md":"Van za 22 april 2023 tot zo 10 september 2023"
            }
         },
         ... // abbreviated for readabillity
      }
   }
]
```